### ISSUE TITLE
Add Error Handling in the frontend 

### ISSUE THREAD

**Issue by shrinish123** [shrinish123/lets-Upsolve#18]
So right now whenever we make an API request , and if we might face an error the only thing done is that we console.log the error , which is bad with respect to user perspective. I would suggest to have a npm package and to show errors properly if anything goes wrong

**Comment by MaYaNkKashyap681** [comment]
can i work on this issue?

**Comment by shrinish123** [comment]
Yes you can go ahead, would recommend you to use [React Toastify](https://www.npmjs.com/package/react-toastify) to show the errors and also success messages if any. don't show success messages like data got loaded etc, only if there's error fetching data you can  say something like Something went wrong, etc. 
Also if you get success response we need to check if the status is ok or failed. Like lets say someone enters a nonexistent codeforces handle it will return a successful response as shown in the image below : 

![erorrhandling ](https://user-images.githubusercontent.com/56491104/207456166-584a2777-fe7f-450c-9e26-4a39597557e6.png)

So what we need to check is also if the status is ok or failed, (even after we get a successful response )  and if the status is failed add a error message where the message can be simply the same that we get from the response as shown in the above image

**Comment by shrinish123** [comment]
I would assign this to you, just please give me a date for your 1st PR submission, it doesn't need to complete implementation, just a small part to track your progress and that would be good.
